app/assets/javascripts/discourse/tests/acceptance/user-preferences-sidebar-test.js

```diff
@@ -52,6 +52,17 @@ acceptance("User Preferences - Sidebar", function (needs) {
     });
   });
 
+  test("sidebar preferences link is not shown when navigation menu is set to legacy", async function (assert) {
+    this.siteSettings.navigation_menu = "legacy";
+
+    await visit("/u/eviltrout/preferences");
+
+    assert.notOk(
```

Comment:
In case you don't know, you can write it like this:

```
assert.dom(".nav-sidebar").doesNotExist();
```

Reply:
Nice :+1: I did not know about this.

Reply:
Full list is available here: https://github.com/mainmatter/qunit-dom/blob/master/API.md